### PR TITLE
writer: Write now returns backup size

### PR DESCRIPTION
**Pull request by fanminshi**
Write() should return the size of written file. 
The size is required in backup manager to construct backup status.

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

**Comment by hongchaodeng** [comment]
lgtm

**Comment by fanminshi** [comment]
merge when green.

File: pkg/backup/writer/s3_writer.go [hunk 1]
```diff
@@ -25,26 +26,41 @@ import (
...
 			Bucket: aws.String(bk),
 			Key:    aws.String(key),
 			Body:   r,
 		})
-	return err
+	if err != nil {
+		return 0, err
+	}
+
+	resp, err := s3w.s3.ListObjects(&s3.ListObjectsInput{
```
Comment (hongchaodeng) [inline pkg/backup/writer/s3_writer.go]: Use GetObject: http://docs.aws.amazon.com/sdk-for-go/api/service/s3/#GetObjectOutput

**Comment by fanminshi** [comment]
all fixed. PTAL @hongchaodeng

**Pull request merged**